tests/avify4mtest.c

```diff
@@ -158,14 +158,15 @@ static avifBool encodeDecodeY4m(uint32_t width,
 
 //------------------------------------------------------------------------------
 
-int main(int argc, char * argv[])
+int main(void)
 {
-    if (argc != 2 || !strlen(argv[1])) {
-        fprintf(stderr, "Missing temporary folder path argument\n");
+    const char * testTmpdir = getenv("TEST_TMPDIR");
```

Comment:
Using the hardcoded environment variable name "TEST_TMPDIR" optimizes for the Bazel and Blaze build environments.

We probably should use our own environment variable name, such as "AVIF_TEST_TMP_DIR". That will require passing the environment variable name as a command-line argument to avify4mtest.c. What do you think?

Reply:
Yes, passing the environment variable name as a command-line argument to avify4mtest is a good idea.

Reply:
Done.